code/__DEFINES/reactions.dm

```diff
@@ -176,15 +176,17 @@
 #define REACTION_OPPRESSION_THRESHOLD 5
 
 // Halon:
-/// The minimum temperature required for halon to form from tritium and BZ.
-#define HALON_FORMATION_MIN_TEMPERATURE 30
-/// The maximum temperature required for halon to form from tritium and BZ.
-#define HALON_FORMATION_MAX_TEMPERATURE 55
-/// The amount of energy 4.25 moles of halon forming from tritium and BZ releases.
-#define HALON_FORMATION_ENERGY 300
+/// Energy released per mole of BZ consumed during halon formation.
+#define HALON_FORMATION_ENERGY 91232.1
 
 /// How much energy a mole of halon combusting consumes.
 #define HALON_COMBUSTION_ENERGY 2500
+/// The minimum temperature required for halon to combust.
+#define HALON_COMBUSTION_MIN_TEMPERATURE (T0C + 70)
+/// The number of moles combusted required to release resin.
+#define HALON_COMBUSTION_MINIMUM_RESIN_MOLES 0.09
```

Comment:
similar

Reply:
Wanted to give some room for halon to react before plasma reaches combustion temperature, so it's better at trapping a spreading plasma fire.

Halon combustion consumes halon slowly, so I chose a rather small value for the requirements, without having like 1 fempto mole of halon release resin.

Reply:
I mean like why is it not 0.1 why is it 0.09

Reply:
Oh I remember. 0.1 moles would be too much for the combustion rate near the minimum temperature.

Reply:
I composed the define with other defines so the reasoning should be more obvious now.